Cubical/Structures/Group/Base.agda

```diff
@@ -163,3 +179,66 @@ makeGroup-left {A = A} id comp inv set assoc lUnit lCancel =
           ≡⟨ lUnit a ⟩
         a
           ∎
+
+
+open Group
+open IsGroup
+
+open Group
+η-Group : {G H : Group {ℓ}}
+       → (p : ⟨ G ⟩ ≡ ⟨ H ⟩)
+       → (q : PathP (λ i → p i) (0g G) (0g H))
+       → (r : PathP (λ i → (p i) → (p i) → (p i)) (_+_ G) (_+_ H))
+       → (s : PathP (λ i → p i → p i) (-_ G) (-_ H))
+       → PathP (λ i → IsGroup (q i) (r i) (s i)) (isGroup G) (isGroup H)
+       → G ≡ H
+Carrier (η-Group p _ _ _ _ i) = p i
+0g (η-Group _ q _ _ _ i) = q i
+_+_ (η-Group _ _ r _ _ i) = r i
+- η-Group _ _ _ s t i = s i
+isGroup (η-Group _ _ _ _ t i) = t i
+
+isSetCarrier : ∀ {ℓ} → (G : Group {ℓ}) → isSet ⟨ G ⟩
+isSetCarrier G = IsSemigroup.is-set (IsMonoid.isSemigroup (isMonoid (isGroup G)))
+
+open import Cubical.Foundations.HLevels
+open IsMonoid
+open IsSemigroup
+dirProd : ∀ {ℓ ℓ'} → Group {ℓ} → Group {ℓ'} → Group
+Carrier (dirProd A B) = Carrier A × Carrier B
+0g (dirProd A B) = (0g A) , (0g B)
+_+_ (dirProd A B) a b = (_+_ A (fst a) (fst b)) , _+_ B (snd a) (snd b)
+-_ (dirProd A B) a = (-_ A (fst a)) , (-_ B (snd a))
+is-set (isSemigroup (isMonoid (isGroup (dirProd A B)))) =
+  isOfHLevelΣ 2 (isSetCarrier A) λ _ → isSetCarrier B
+assoc (isSemigroup (isMonoid (isGroup (dirProd A B)))) x y z i =
+  assoc (isGroup A) (fst x) (fst y) (fst z) i , assoc (isGroup B) (snd x) (snd y) (snd z) i
+identity (isMonoid (isGroup (dirProd A B))) x =
+   (λ i → IsGroup.rid (isGroup A) (fst x) i , IsGroup.rid (isGroup B) (snd x) i)
+ , λ i → IsGroup.lid (isGroup A) (fst x) i , IsGroup.lid (isGroup B) (snd x) i
+inverse (isGroup (dirProd A B)) x =
+    (λ i → (fst (inverse (isGroup A) (fst x)) i) , (fst (inverse (isGroup B) (snd x)) i))
+  , λ i → (snd (inverse (isGroup A) (fst x)) i) , (snd (inverse (isGroup B) (snd x)) i)
+
+open import Cubical.Data.Unit
+trivialGroup : Group₀
+Carrier trivialGroup = Unit
+0g trivialGroup = _
+_+_ trivialGroup _ _ = _
+-_ trivialGroup _ = _
+is-set (isSemigroup (isMonoid (isGroup trivialGroup))) = isSetUnit
+assoc (isSemigroup (isMonoid (isGroup trivialGroup))) _ _ _ = refl
+identity (isMonoid (isGroup trivialGroup)) _ = refl , refl
+inverse (isGroup trivialGroup) _ = refl , refl
+
+
+open import Cubical.Data.Int renaming (_+_ to _+Int_ ; _-_ to _-Int_)
```

Comment:
Move up the import. We should probably move this to Data.Int as well, but let's do it in a separate PR